client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -351,11 +370,7 @@ function CompatCell({
 }) {
   const supportClassName = getSupportClassName(support);
   const browserReleaseDate = getSupportBrowserReleaseDate(support);
-  const hasNotes =
-    support &&
-    asList(support).some(
-      (item) => item.prefix || item.notes || item.alternative_name || item.flags
-    );
+  const hasNotes = support && asList(support).length > 1;
```

Comment:
Can you put some of the description you put into the PR description here as a code comment.
It stands out like a rather "odd" piece of business logic. It might make perfect sense inside the BCD project but perhaps it would be less surprising if we can write someone down here to remind the Yari developer looking at this.

Reply:
I think it is odd that this was named "hasNotes". It should rather be "hasMultipleSupportStatements" or so.
I added a comment.

Reply:
I can kinda understand why it was called `hasNotes`. 
On https://developer.mozilla.org/en-US/docs/Web/CSS/appearance#browser_compatibility you can click to expand...
<img width="1218" alt="Screen Shot 2021-05-24 at 3 13 57 PM" src="https://user-images.githubusercontent.com/26739/119396341-bb6ae200-bca2-11eb-9d62-8405e73a3fb5.png">
...to be able to see and read about what the "-x-" thing means.

Reply:
After all, the button has a tooltip that is "Open implementation notes". 

But if you think it's not valuable to be able to expand those that now cease to be expandable, then I trust you.

Reply:
Sounds like in addition to multiple support statements, the "show more stuff" arrow should be shown if there are notes for a single entry too?

And the tooltip will now be wrong in some cases, should it depend on whether there are notes, or always be "show details" or something?

Reply:
> Sounds like in addition to multiple support statements, the "show more stuff" arrow should be shown if there are notes for a single entry too? 

Oh yes, thanks, I was focusing too much on the issue at hand. The original check for hasNotes needs to stay but it was incomplete. Added this back.

> And the tooltip will now be wrong in some cases, should it depend on whether there are notes, or always be "show details" or something?

Not sure what you mean exactly but I noted the notes were a bit off, so I fixed them. The order of the icons and the corresponding notes should now align. 
Sometimes BCD provides notes that say the same thing that is already implied by data, but I guess that's more of a data issue.